### PR TITLE
Show host graph

### PR DESCRIPTION
**Pull request by liuyu85cn**
please refers to https://github.com/vesoft-inc/nebula-graph/pull/437

### CODE REVIEW

**Review by critical27 — DISMISSED** [review]

**Review by panda-sheep — COMMENTED** [review]
LGTM

**Review by panda-sheep — APPROVED** [review]
LGTM

File: src/common/interface/meta.thrift [hunk 1]
```diff
@@ -474,8 +474,11 @@ struct ListEdgesResp {
 }
 
 enum ListHostType {
-    ALLOC       = 0x00,
     // nebula 1.0 show hosts, show leader, partition info
+    ALLOC       = 0x00,
+    GRAPH       = 0x01,
+    META        = 0x02,
+    STORAGE     = 0x03,
```
Comment (laura-ding) [inline src/common/interface/meta.thrift:481]: The ListHostsReq is no longer need ` role`? delete the role of  ListHostsReq

Comment (liuyu85cn) [reply src/common/interface/meta.thrift:481]: fixed.

**Review by laura-ding — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Pull request merged**